src/routers/controllers/createGroupController.ts

```diff
@@ -12,7 +13,11 @@ export const createGroupController = async ({ body: groupDTO }: Request, res: Re
 
         res.status(200).send({ message: `Group ${groupDTO.id} created successfully` });
     } catch (error) {
-        console.error(error);
+        controllerErrorHandler({
+            methodName: createGroupController.name,
+            methodArguments: { body: groupDTO },
+            error
+        });
         res.json({ error });
```

Comment:
You should explicitly set a response status or pass the error to the global error handler.